lib/cli/commands/listpairs.ts

```diff
@@ -1,11 +1,34 @@
 import { callback, loadXudClient } from '../command';
 import { Arguments } from 'yargs';
-import { ListPairsRequest } from '../../proto/xudrpc_pb';
+import { ListPairsRequest, ListPairsResponse } from '../../proto/xudrpc_pb';
+import Table, { HorizontalTable } from 'cli-table3';
+import colors from 'colors/safe';
+import { getPairId } from '../../utils/utils';
+
+const HEADERS = [colors.red('PairId')];
```

Comment:
This is subjective and minor, but I would propose a different color for our standard header rather than the red that we use for the "sell" columns elsewhere. Again I'd suggest blue or dark blue, but I'm not too picky.